tuple/tuple.mbti

```diff
@@ -26,4 +28,15 @@ fn uncurry[T, U, V]((T) -> (U) -> V) -> (T, U) -> V
 // Traits
 
 // Extension Methods
+impl @arbitrary.Arbitrary for Tuple
```

Comment:
@CAIMEOX let's call the package quickcheck and refer it as @quickcheck.Arbitrary here